packages/serverless/src/domain/use-case/group.ts

```diff
@@ -149,23 +154,33 @@ export class Group {
   /**
    * Updates the metadata of a group based on the provided parameters.
    *
-   * @param params - The parameters specifying the group and the metadata to update, following the `TGroupParamUpdateMetadata` type.
+   * @param paramUpdateMetadata - The parameters specifying the group and the metadata to update, following the `TGroupParamUpdateMetadata` type.
    * @param session - (Optional) The MongoDB session for transactional queries.
    * @returns A promise resolving to `true` if the metadata was updated successfully, otherwise `false`.
    */
   public static async updateMetadata(
-    params: TGroupParamUpdateMetadata,
+    paramUpdateMetadata: TGroupParamUpdateMetadata,
     session?: ClientSession
   ): Promise<boolean> {
-    const { databaseName, groupName, createdBy, newGroupName, newDescription } =
-      params;
+    const {
+      databaseName,
+      groupName,
+      createdBy,
+      newGroupName,
```

Comment:
`newGroupName` and `newGroupDescription` are both optional fields.

Reply:
It's already optional refer to param type

```ts
type TGroupParamUpdateMetadata = Pick<TGroupParam, "databaseName" | "createdBy" | "groupName"> & {
    newGroupName?: string;
    newGroupDescription?: string;
}
```